js/modules/k6/k6.go

```diff
@@ -182,6 +184,13 @@ func (*K6) Check(ctx context.Context, arg0, checks goja.Value, extras ...goja.Va
 				succ = false
 			}
 		}
+
+		// Fail if a single iteration caused an error
+		// This is kept for backwards compatibility, but should be removed
```

Comment:
I don't think this comment is needed here, that new `check()` API would be something completely separate, likely with a completely different name and/or import path, so this comment would probably end up stale and forgotten in time :smile:

Reply:
OK, I'll remove it.